feat: batect manager (#7898)
renovatebot/renovate: lib/manager/batect/readme.md

Extracts all Docker images from Batect configuration files.

> By default, this searches for files called `batect.yml` or `batect-bundle.yml`. If you use other file names for your
> Batect configuration, you'll need to configure regular expressions for them in `renovate.json`:

Reviewer: ```suggestion
By default, the manager searches for files called `batect.yml` or `batect-bundle.yml`.
If you keep your Batect configuration in another file, you'll need to tell Renovate where to find the Batect file.

You do this by creating a `"batect"` object in the `renovate.json` file, this object contains a `fileMatch` array, which has a regular expression that matches the configuration filename.

E.g.:
```

Extracts all Docker images from Batect configuration files.

By default, this searches for files called `batect.yml` or `batect-bundle.yml`. If you use other file names for your
Batect configuration, you'll need to configure regular expressions for them in `renovate.json`:

```json
{
  "batect": {
    "fileMatch": [

>       "(^|/)batect\\.yml$",
>       "(^|/)batect-bundle\\.yml$",

Reviewer 2: ```suggestion
      "(^|/)batect(-bundle)?\\.yml$",
```

Extracts all Docker images from Batect configuration files.

By default, the manager searches for files called `batect.yml` or `batect-bundle.yml`.

> If you keep your Batect configuration in other files, you'll need to tell Renovate where to find them. This includes files included into your main configuration file with `include`.

Reviewer: ```suggestion
If you keep your Batect configuration in other files, you'll need to tell Renovate where to find them.
This includes files included into your main configuration file with `include`.
```

> You do this by creating a `"batect"` object in your `renovate.json` file. This object should contain a `fileMatch` array with regular expressions that match the configuration file names.

Reviewer: ```suggestion
You do this by creating a `"batect"` object in your `renovate.json` file.
This object should contain a `fileMatch` array with regular expressions that match the configuration file names.
```
